packages/asc-ui/src/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent, HTMLAttributes } from 'react'
+import { ChevronRight } from '@amsterdam/asc-assets'
+import BreadcrumbsItemStyle, {
+  liProps,
+  StyledIcon,
+} from './BreadcrumbsItemStyle'
+
+const BreadcrumbsItem: FunctionComponent<
+  liProps & HTMLAttributes<HTMLLIElement>
+> = ({ children, href, className, ...otherProps }) => (
+  <BreadcrumbsItemStyle className={className} {...otherProps}>
+    <StyledIcon size={7}>
```

Comment:
The Icon seems to be larger in the design system. 
![Screen Shot 2020-10-15 at 16 04 05](https://user-images.githubusercontent.com/1895171/96140999-dd2c0b80-0f00-11eb-82ee-638fa761a619.png)

Reply:
ik heb er 6 van gemaakt: in het DS is het icon ook nog  vetter. daar moet de stijl politie maar naar kijken

Reply:
In het design system is hij juist groter :-) 10 oid ?

Reply:
hier zie je dat het 6 pixel hoog is in het design:

<img width="436" alt="height of chev icon " src="https://user-images.githubusercontent.com/4640157/96433399-7ad75180-1205-11eb-8beb-0c6adf760ed8.png">

Reply:
Ik heb 't idee dat dat niet de goede referentie is. Onder dat design staan de atoms en sketch voorbeelden.

https://designsystem.amsterdam.nl/7awj1hc9f/p/35ca97-breadcrumbs/i/6632435

Misschien moet even nagevraagd worden bij designsystem mensen welke versie de waarheid is.

Reply:
ja daar is hij veel groter

Reply:
het is gelukt:
<img width="520" alt="Screenshot 2020-10-19 at 13 50 06" src="https://user-images.githubusercontent.com/4640157/96447337-0951d000-1212-11eb-85b7-a0b9c8769ff0.png">